src/main/resources/test_files/runtime/FunctionObjectDescendant.iq

```diff
@@ -0,0 +1,4 @@
+(:JIQS: ShouldRun; Output="( { "a" : [ 1, { "b" : 2 } ], "o" : { "c" : 3 } }, { "b" : 2 }, { "c" : 3 } )" :)
```

Comment:
We need tests on corner cases (top-level array, empty object, empty array, list of atomics, etc)